### PR TITLE
Feature ui

### PR DESCRIPTION
**Pull request by sachind-cuelogic**
- Dashboard UI
- Project list UI

### CODE REVIEW

File: bms/bms_app/static/bms_app/css/create_project.css [hunk 1]
```diff
@@ -4,9 +4,39 @@
 }
 
 .panel-body {
-    padding: 10px 10px 0px 10px;
+	padding: 10px 10px 0px 10px;
```
Comment (nileshm-cuelogic) [inline bms/bms_app/static/bms_app/css/create_project.css]: In shorthand properties 10 px(top) 10px (right) 0px(bottom) 10px (left); if right and left value is equal then no need to mention left value. eg 10px 10px 0; is sufficient,

File: bms/bms_app/static/bms_app/css/create_project.css [hunk 2]
```diff
@@ -4,9 +4,39 @@
...
 }
 
+.page-header {
+	padding-bottom: 9px;
+	margin: 20px 0 20px;
```
Comment (nileshm-cuelogic) [inline bms/bms_app/static/bms_app/css/create_project.css]: In shorthand properties 20 px(top) 0 (right) 20px(bottom) 0 (left); if top and bottom, right and left values are equal then no need to mention bottom and left. eg 20px 0; is sufficient.

File: bms/bms_app/static/bms_app/css/create_project.css [hunk 3]
```diff
@@ -63,33 +79,39 @@
...
 .remove{
-    margin-right: 10px;
+	margin-right: 10px;
+	vertical-align: top;
 }
 
 .glyphicon-remove{
 	font-size: 14px;
-	top: 0px;
+	top: 0;
```
Comment (nileshm-cuelogic) [inline bms/bms_app/static/bms_app/css/create_project.css]: top: 0; and right: 0; properties will work only with position property, without position property they don't make any difference in CSS

File: bms/templates/bms_app/register.html [hunk 4]
```diff
@@ -1,10 +1,11 @@
 {% extends "registration/common_login_reg.html" %}
+{% block title %}Registration Form{% endblock %}
 {% load static %}
 {% block cssfile %}
 <link rel="stylesheet" type="text/css" href="{% static 'bms_app/css/common_login_layout.css' %}">
 {% endblock %}
 {% block content %}
-<div id="register-box" class="mainbox col-md-4 col-md-offset-4">
+<div id="register-box" class="mainbox col-md-4 col-md-offset-4 col-sm-4 col-sm-offset-4">
```
Comment (nileshm-cuelogic) [inline bms/templates/bms_app/register.html]: if you are using col-sm-4 then no need to add col-md-4. and col-md-offset-4

File: bms/bms_app/static/bms_app/js/project_list.js [hunk 5]
```diff
@@ -57,3 +57,10 @@ $(document).ready(function() {
     });
 });
 
+$(document).ready(function() {
```
Comment (tusharm-cuelogic) [inline bms/bms_app/static/bms_app/js/project_list.js]: no need to use "+$(document).ready(function() {" each and every time if already used, you can put event in the existing one

**Comment by tusharm-cuelogic** [comment]
resolved JS comments

**Pull request merged**